### ISSUE TITLE
NFS example will break due to default dynamic provisioning

### ISSUE THREAD

**Issue by jingxu97** [kubernetes/kubernetes#39459]
The NFS example under https://github.com/kubernetes/kubernetes/tree/master/examples/volumes/nfs has an issues

nfs-pvc will not be able to bound to manually created nfs-pv object after release 1.6 with default storageclass. In PR #31617, a default storageclass is deployed automatically. Because of this change, for persistentvolumeclaim that does not have any storageclass annotation (as in https://github.com/kubernetes/kubernetes/blob/master/examples/volumes/nfs/nfs-pvc.yaml), default storageclass will automatically create a GCE-pd persistent volume object to bound t 
  he claim. So nfs-pvc will not bound to the nfs-pv object created manually. 
To solve this issue, there are a few possible ways, but they are not reliable based on my experiences. 
1. delete storageclass API object. I tried a number of times, most of time it does not work because storageclass is recreated whenever a pvc is created. Sometimes, pvc is in pending state for a long time. 
2. edit storageclass API object: to set is-default-class to false. After this update, a new created pvc is in pending state for long time.

**Comment by thockin** [comment]
You can set the class to "", which should force-select only PVs that have
class = ""

**Comment by msau42** [comment]
I did #2 and it worked.  Since the nfs pv example uses "retain" as the reclaimpolicy, I had to manually delete the pv and recreate the pv to make it available.

**Comment by msau42** [comment]
I also tried the "" class suggestion, which also worked, but would require users to use a beta feature to have this behavior.

```
annotations:
   volume.beta.kubernetes.io/storage-class: ""
```

**Comment by copejon** [comment]
@msau42 It seems that with the latest k8s code, dynamic provisioning / default storage class is enabled by default.   Would updating the example to reflect this (adding the "" class annotation) be the right way to solve this?

**Comment by msau42** [comment]
Since storageclass is supposed to go to GA this release, I think that is fine.  We could change the example to use the beta annotation now, and change it to the official annotation once that change is made, or wait until we have the official annotation.

**Comment by msau42** [comment]
This has been fixed in kubernetes/examples#30
